osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -146,26 +176,39 @@ func addPackageInstallRemovalTest(pkgTestSetup []*packageManagementTestSetup) []
 	for _, pkgManager := range pkgManagers {
 		var oc *osconfigpb.OsConfig
 		var assign *osconfigpb.Assignment
-		var instaneName, ss, vs string
+		var instanceName, image, ss, vs string
 
 		switch pkgManager {
 		case "apt":
-			instaneName = fmt.Sprintf("%s-%s", filepath.Base(debianImage), testName)
+			image = debianImage
+			instanceName = fmt.Sprintf("%s-%s", filepath.Base(image), testName)
```

Comment:
same as above

Reply:
changed. and also moved out some other common code.